plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/editors/DateTimeInlineEditor.java

```diff
@@ -120,8 +121,9 @@ public void run() {
                 }
             }
             try {
-                if (!(parent.extractEditorValue() instanceof Date)){
-                    UIUtils.asyncExec(() -> DBWorkbench.getPlatformUI().showWarningMessageBox(ResultSetMessages.dialog_value_view_error_parsing_date_title, ResultSetMessages.dialog_value_view_error_parsing_date_message));
+                Object value = parent.extractEditorValue();
+                if (!(value instanceof Date)){
+                    UIUtils.asyncExec(() -> DBWorkbench.getPlatformUI().showWarningMessageBox(ResultSetMessages.dialog_value_view_error_parsing_date_title, NLS.bind(ResultSetMessages.dialog_value_view_error_parsing_date_message, value)));
```

Comment:
This should be an error, not a warning. See https://github.com/dbeaver/dbeaver/pull/15232#issuecomment-1020470245